db/migrate/20180909203857_drop_authors_to_sort_on.rb

```diff
@@ -0,0 +1,5 @@
+class DropAuthorsToSortOn < ActiveRecord::Migration[5.1]
```

Comment:
We usually use "remove" rather than "drop" when naming migrations. If you could update that, it would be great, but it's not something that will stop us from merging this change.